### ISSUE TITLE
the '-b' flag is not being respected in v3

### ISSUE THREAD

**Issue by sfabriece** [ionic-team/ionic-cli#2215]
#### What version of the CLI are you using? Output of `ionic --version`

3.0.0-rc.1

#### Short description of the problem:

the following command opens the browser.

`ionic serve -scb --lab`

#### What behavior are you expecting?

I expect the command to run without a browser tab being opened.

**Post the output of `ionic info` below please**
```
global packages:

    @ionic/cli-utils : 1.0.0-rc.1
    Ionic CLI        : 3.0.0-rc.1

local packages:

    @ionic/app-scripts              : 1.3.7
    @ionic/cli-plugin-ionic-angular : 1.0.0-rc.1
    Ionic Framework                 : ionic-angular 3.1.1

System:

    Node       : v7.8.0
    OS         : Windows 10
    Xcode      : not installed
    ios-deploy : not installed
    ios-sim    : not installed

```

**Other information:** (e.g. stacktraces, related issues, suggestions how to fix, stackoverflow links, forum links, etc)

output after running the command.

`Starting app-scripts server: --consolelogs --serverlogs --nobrowser --lab --port 8100 --livereload-port 35729 --address 0.0.0.0`

**Comment by dwieeb** [comment]
Right now `--lab` overrides `-b`. It doesn't make sense to supply both flags, as `--lab` is a feature for the launched browser.

**Comment by sfabriece** [comment]
since I can always go the 'lab' using the url '/ionic-lab', I expect passing -b to mean, I explicitly don't want to open the browser because I know which Urls to go to.

**Comment by dwieeb** [comment]
Gotcha @sfabriece, I understand 👍  

@danbucholtz Can we review this line here: https://github.com/driftyco/ionic-app-scripts/blob/c17e6dfa132039e92c6cc3042a38183ffeb531dc/src/serve.ts#L76

**Comment by dwieeb** [comment]
@sfabriece Probably won't be fixed for v3 launch, but we'd like to fix this in the future.

**Comment by sfabriece** [comment]
sounds good @dwieeb. Should i close this issue?

**Comment by dwieeb** [comment]
@sfabriece If you don't mind I'll keep it open under the milestone.

**Comment by dwieeb** [comment]
Hi @sfabriece, I finally got around to this. It will be fixed in 3.7.

**Comment by sfabriece** [comment]
awesome @dwieeb
